### ISSUE TITLE
rails db:migrate in a clean db doesn't work

### ISSUE THREAD

**Issue by davidsf** [opensouthcode/moraga#1]
**I'm submitting a ..**
- [X] Bug Report
- [ ] Feature Request

**Current behavior:**

rails db:migrate in a clean repository breaks:

```
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Active Record release the migration was written for:

  class DeviseCreateUsers < ActiveRecord::Migration[7.0]
/home/zoconet/src/moraga/db/migrate/20121223110942_devise_create_users.rb:3:in `<top (required)>'

Caused by:
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Active Record release the migration was written for:

  class DeviseCreateUsers < ActiveRecord::Migration[7.0]
```

**Expected correct behavior:**

It must run correctly.

**Steps to reproduce:**

Just configure the development env and try to run `rails db:migrate RAILS_ENV=developemnt`

**Other information:**

After this first error, there are others than do that the run migration imposible.

**Comment by davidsf** [comment]
There are migrations for creation of tables that and after there are migrations that drops it:

```
targets
visits
ahoy_events
campaigns
```

**Comment by davidsf** [comment]
Fixed by PR #2
